File: week-02/day-02/summing.py
# - Write a function called `sum` that sum all the numbers
#   until the given parameter


numbers = [1, 2, 3, 4, 5]
def sum (numbers):
    num_sum = 0
    for i in numbers:
        num_sum += i
    return num_sum
    
print(sum(numbers))

File: week-07/day-02/summing.js
'use strict';
// - Write a function called `sum` that sum all the numbers until the given parameter
// - The function should return the result

let num = 5

function sum (num) {
  let addNum = 0;
  for (let i = 0; i <= num; i++) {  
    addNum += i;
    
  }
  return addNum;
}

console.log(sum(num))